.github/workflows/e2e-conformance-trigger.yaml

```diff
@@ -1,19 +1,39 @@
 name: E2EConformanceTrigger
 on:
   schedule:
-    - cron: '7 22 * * *'
+    - cron: '7 22 1,15 * *'
   workflow_dispatch:
 jobs:
+  resolve-git-ref:
```

Comment:
Do we need this since a workflow_run will never trigger this? That is -- until we decide to support an action like `/karpenter conformance`